android/build.rs

```diff
@@ -0,0 +1,11 @@
+// Copyright 2024 The ChromiumOS Authors
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+//! cros-libva bindings generator in Android.
+
+include!("../lib/bindgen.rs");
```

Comment:
Using `include!()` macro is not consider a good practice for source code that not generated.

I think 
```
#[path = "file.rs"]
mod bindgen_gen;
````
would be a better solution.